src/utils/string/index.ts

```diff
@@ -52,9 +51,8 @@ export const msToDuration = (ms: number): string => {
     return `${minutes}m ${seconds}s`;
   }
   if (seconds > 0) {
-    return `${seconds}s ${milliseconds}ms`;
+    return `${seconds}s`;
```

Comment:
im just curious, but what was the reason for this change?

Reply:
The ms line has always been a pet peeve of mine tbh 😅 not useful and honestly makes the time harder to parse imo since the ms value is somewhat random

Reply:
It's also a pet peeve of mine but maybe it's required for durations since test run times are captured in ms. We can disable this for normal times

Reply:
It's no problem to just omit the millisecond remainder that's calculated here. Here's a running task, you can see that we just do a countdown in seconds once the time is < 1min. Lmk if you meant something else by this comment though!

https://user-images.githubusercontent.com/9298431/198697426-5b6906b9-c0b2-4676-8618-1bb0432d074b.mov